lib/tacos/conflict_detection.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3.12
+from __future__ import annotations
+
+from lib import json
+from lib import wait
+from lib.functions import one
+from lib.parse import Parse
+from lib.sh import sh
+from lib.types import URL
+from lib.user_error import UserError
+
+JOB = "conflict-detection"
+TAG = f'<!-- getsentry/tacos-gha "{JOB}" -->'
+MERGE_CONFLICT_MESSAGE = f"""\
+You currently have a merge conflict, which prevents github-actions from running
+any checks. TACOS will resume doing terraform stuff for you once the conflict is
+resolved.
+{TAG}
+"""
+
+# mypy can't find LessThanOneError
```

Comment:
Is this still applicable? LessThanOneError doesn't appear in this file?

Reply:
You're right; fixed.